### ISSUE TITLE
Please add technical details to the README

### ISSUE THREAD

**Issue by tejas-gokhale** [ASU-Active-Perception-Group/BIRD_Code#1]
Hi @shailaja183 

Can you add the following things to the readme/repo?

- [ Train-test-validation --split and size ] 
- [ Quantitative Results -- accuracies ] 
- [ A description of the method -- input-output, loss function, q-function, reward function ...  ] 
- [ A block diagram of the system ] 
- [ Qualitative Results -- a few examples of input pairs and plan generated vs ground truth plan ] 

I will use this for writing the paper.
Thanks,
T.
